service/storage_service.cc

```diff
@@ -443,23 +443,7 @@ future<> storage_service::topology_state_load(cdc::generation_service& cdc_gen_s
     }
 }
 
-future<> storage_service::topology_transition(storage_proxy& proxy, cdc::generation_service& cdc_gen_svc, gms::inet_address from, std::vector<canonical_mutation> cms) {
-    assert(this_shard_id() == 0);
```

Comment:
I think this assertion should stay. It's still true that `topology_transition` needs to be called on shard 0.

Reply:
Fixed.